NetworkDispatcherKtor/src/commonMain/kotlin/com/sdk/growthbook/network/GBNetworkDispatcherKtor.kt

```diff
@@ -109,6 +118,7 @@ class GBNetworkDispatcherKtor(
         CoroutineScope(PlatformDependentIODispatcher).launch {
             try {
                 prepareGetRequest(url).execute { response ->
+                    eTag = response.headers["ETag"]
```

Comment:
This SDK makes 2 network calls.  The first to `/api/features/sdk-abcdef` to download the payload.  The second to `/sub/sdk-abc123` to establish a streaming SSE connection.  We only care about etags for the first request

Reply:
if multiple URLs are used, then we'll have to change the implementation. I'll fix that!